components/CheckoutPage.tsx

```diff
@@ -27,11 +27,15 @@ const CheckoutPage = () => {
       setGst(found.gst);
       setCurrency(found.currency);
       setPin(found.postal);
-      setTotal(found.value * found.gst + found.value);
```

Comment:
get email from query param of the URL like 
https://www.prodready.dev/checkout?email=souvikbasu@gmail.com

Reply:
const [formData, setFormData] = useState({
    email: "",
    country: "US",
    postcode: "",
  });

Load the initial settings for US